packages/blade/src/components/QuickFilters/docs/QuickFilter.stories.tsx

```diff
@@ -0,0 +1,131 @@
+      apiDecisionLink={null}
+      figmaURL="https://www.figma.com/design/jubmQL9Z8V7881ayUD95ps/Blade-DSL?node-id=100413-32686&t=n9A7LztwEkIsly3v-0"
+    >
+      <Heading size="large">Usage</Heading>
+      <Sandbox showConsole>
+        {`
+        import { QuickFilterGroup,QuickFilter  } from '@razorpay/blade/components';
+        
+        function App() {
+          return (
+            <QuickFilterGroup >
+             <QuickFilter title="Title1" value="value1" trailingElement={<Counter value={234} />} />{' '}
+             <QuickFilter title="Title2" value="value2" trailingElement={<Counter value={234} />} />{' '}
+           </QuickFilterGroup>
+          )
+        }
+
+        export default App;
+      `}
+      </Sandbox>
+    </StoryPageWrapper>
+  );
+};
+
+export default {
+  title: 'Components/QuickFilter & QuickFilterGroup',
+  component: QuickFilterGroup,
+  tags: ['autodocs'],
+  argTypes: {
+    ...getStyledPropsArgTypes(),
+  },
+  parameters: {
+    docs: {
+      page: Page,
+    },
+  },
+} as Meta<QuickFilterGroupProps>;
+
+const QuickFilterTemplate: StoryFn<typeof QuickFilterGroup> = (args) => {
+  return (
+    <QuickFilterGroup {...args}>
+      <QuickFilter
+        title="All"
+        value="All"
+        trailingElement={<Counter value={400} color="information" />}
+      />
+      <QuickFilter
+        title="Captured"
+        value="Captured"
+        trailingElement={<Counter value={234} color="positive" />}
+      />
+      <QuickFilter
+        title="Failed"
+        value="Failed"
+        trailingElement={<Counter value={234} color="negative" />}
+      />
+    </QuickFilterGroup>
+  );
+};
+
+export const Default = QuickFilterTemplate.bind({});
+Default.storyName = 'Default';
+Default.args = {
+  selectionType: 'single',
+};
+
+const QuickFilterSingle: StoryFn<typeof QuickFilterGroup> = () => {
+  return (
+    <QuickFilterGroup selectionType="single">
+      <QuickFilter
+        title="Unresolved"
+        value="unresolved"
+        trailingElement={<Counter value={234} color="information" />}
+      />
+      <QuickFilter
+        title="Resolved"
+        value="resolved"
+        trailingElement={<Counter value={234} color="positive" />}
+      />
+      <QuickFilter
+        title="In Progress"
+        value="in_progress"
+        trailingElement={<Counter value={234} color="neutral" />}
+      />
+    </QuickFilterGroup>
+  );
+};
+
+export const QuickFilterSingleStory = QuickFilterSingle.bind({});
+QuickFilterSingleStory.storyName = 'QuickFilter Single Selection';
+
+const QuickFilterMultiple: StoryFn<typeof QuickFilterGroup> = () => {
+  return (
+    <QuickFilterGroup selectionType="multiple">
+      <QuickFilter
+        title="Captured"
+        value="Captured"
+        trailingElement={<Counter value={234} color="positive" />}
+      />
+      <QuickFilter
```

Comment:
Add an example where wrapping a QuickFilter in a tooltip, just to make sure it works properly.